README.md

```diff
@@ -43,69 +43,102 @@ remotes::install_github("mrc-ide/frogger", upgrade = FALSE)
 
 ## Leapfrog to Frogger glossary
 
-### Stratifications
-
-| Leapfrog      | Frogger                   | Details                                                |
-|---------------|---------------------------|--------------------------------------------------------|
-| NG            | num_genders               | Number of genders                                      |
-| pAG           | age_groups_pop            | Number of age groups in population, 81 for 0 to 80+    |
-| pIDX_FERT     | fertility_first_age_group | First age group index eligible for fertility           |
-| pAG_FERT      | age_groups_fertiity       | Number of ages eligible for fertility                  |
-| pIDX_HIVADULT | hiv_adult_first_age_group | Index of the first age group to be considered an adult |
-| hAG           | age_groups_hiv            | Number of age groups in HIV population                 |
-| hDS           | disease_stages            | Number of disease stages                               |
-| hTS           | treatment_stages          | Number of treatment stages                             |
-
-### Model settings
-
-| Leapfrog           | Frogger             | Details                           |
-|--------------------|---------------------|-----------------------------------|
-| sim_years          | sim_years           | Number of simulation years to run |
-| hiv_steps_per_year | hiv_steps_per_year  |                                   |
-| t_ART_start        | time_art_start      | Time step to start modelling ART  |
-| hAG_SPAN           | hiv_age_groups_span | Array of HIV age group sizes      |
+Frogger naming rules
+
+* `p_` prefix indicates this is age stratified by total single year i.e. same age stratification as total population
+* `h_` prefix indicates this is age stratified by same age stratification as HIV adult population, this might be single
+  year from
+  15+ or could be coarse age groups
+* `hc_` prefix indicates this is age stratified by the HIV child population
+* `idx_` prefix means this is an index
```

Comment:
Might be helpful to give an example here of how this is used. I don't think it is very clear from the description.